CONTRIBUTING.md

```diff
@@ -390,36 +390,29 @@ it will run, the user browser.
 
 ### Running Integration Tests
 
-1.1 Without Plugins
+#### Without Plugins
 
 Run DC/OS UI in testing mode (you have to close `npm start`).
 
 ```sh
 npm config delete externalplugins
-npm run start:testing
+npm start
```

Comment:
why start why not testing?

Reply:
I checked them, one is for cypress saving fixture files causing reloading. This is not the case anymore, so I think we can delete that. The other one is for loading the Config.test instead of the Config.dev and I would argue that can go to the trash as well, this might simplify our setup.